packages/viewport/src/index.js

```diff
@@ -187,3 +187,40 @@ export function subscribeIsDesktop( listener ) {
 export function getWindowInnerWidth() {
 	return isServer ? SERVER_WIDTH : window.innerWidth;
 }
+
+/******************************************/
+/*       Vertical Scroll Experiment       */
+/******************************************/
+
+//TODO: To be refactored using above constants and helper methods
```

Comment:
I am not sure I understand this. Could you clarify which constants and which helper methods? :)

Reply:
I have updated the comment to be more clear